game/services.py

```
@@ -0,0 +1,80 @@
+    # 48 hours in seconds
+    stale_threshold = time.time() - (48 * 3600)
+    
+    deleted_count = 0
+    resigned_count = 0
+    
+    # Iterate over all sessions
+    for session in Session.objects.all():
+        try:
+            session_data = session.get_decoded()
+        except Exception:
+            continue
```

Ruff (0.15.12)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)